com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/NodeDescriptorNodeBuilder.cs

```diff
@@ -83,8 +119,8 @@ public void BuildNode(NodeHandler node, Registry registry)
                 }
             }
 
-            // TODO (Brett) THIS IS WRONG!
-            // TODO (Brett) The fallback type should be determined with the currently selected FD.
+            // TODO (Brett) THIS MIGHT BE WRONG!
+            // TODO (Brett) Should the fallback type should be determined with the currently selected FD?
```

Comment:
FallbackTypeResolver works directly off of the node representation and whatever port connections are there. I think it ends up implicitly using whatever the currently selected FD is _because_ that's the one that was used to setup the ports.

But also-- the `.Any` fields that need to be resolved are resolved from upstream connections and don't have to to have anything to do with the FDs. I believe the behavior here is entirely correct and ought to not need adjustment for multi function.

Reply:
Yes. This has become clear. I've removed the comment.